Rakefile

```diff
@@ -14,4 +14,9 @@ Rake::TestTask.new do |t|
   t.warning = true
 end
 
+desc "Run Rubocop"
+task :rubocop do
+  sh "bundle exec rubocop"
```

Comment:
Can you look at using the [RuboCop-provided Rake integration](https://docs.rubocop.org/rubocop/integration_with_other_tools.html#rake-integration) rather than executing this from the shell?

Reply:
done, I have updated the PR